app/forms/review_delete_form.py

```diff
@@ -0,0 +1,6 @@
+from flask_wtf import FlaskForm
+from wtforms import SubmitField
+
+
+class DeleteReviewForm(FlaskForm):
+    delete = SubmitField('Delete')
```

Comment:
Has this been tested? I'm not sure if it will cause problems when we try submitting data. I don't believe this is on any of the other forms that were made in our starter. If its not necessary, maybe try just changing this to `pass` so the form exists for CSRF checking?

Reply:
No this hasn't been test ahaha... I thought this might be necessary for the CSRF checking.  So you mean just removing line 6 and replacing it with pass so as to have the form available for the form.validate_on_submit() in the backend?

Reply:
Having replaced the line with "pass" the delete route works thru Postman. Haven't been able to manage to get the post route to work, however.